components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -482,25 +433,22 @@ public void execute() {
      * @param version API version
      */
     private void validateAPIGetRequestParams(String label, String apiName, String version) {
-        if ((StringUtils.isEmpty(label) && (StringUtils.isEmpty(apiName) || StringUtils.isEmpty(version))) ||
-                StringUtils.isNotEmpty(label) && (StringUtils.isNotEmpty(apiName) || StringUtils.isNotEmpty(version)) ||
-                (StringUtils.isEmpty(apiName) && StringUtils.isNotEmpty(version)) ||
-                (StringUtils.isNotEmpty(apiName) && StringUtils.isEmpty(version))) {
+        if ((StringUtils.isEmpty(label) && (StringUtils.isEmpty(apiName) || StringUtils.isEmpty(version)))
+                || StringUtils.isNotEmpty(label) && (StringUtils.isNotEmpty(apiName) || StringUtils.isNotEmpty(version))
+                || (StringUtils.isEmpty(apiName) && StringUtils.isNotEmpty(version)) || (StringUtils.isNotEmpty(apiName)
+                && StringUtils.isEmpty(version))) {
```

Comment:
Check for line length

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678